build.gradle

```diff
@@ -179,6 +179,13 @@ dependencies {
 
 
     generatorsImplementation("com.squareup:javapoet:1.13.0")
+
+    // Adventure, for text messages
+    api platform("net.kyori:adventure-bom:4.5.1")
+    api "net.kyori:adventure-api"
+    implementation "net.kyori:adventure-text-serializer-gson"
+    implementation "net.kyori:adventure-text-serializer-plain"
+    implementation "net.kyori:adventure-text-serializer-legacy"
```

Comment:
It might be worth exposing these - they are exposed as API with Sponge, Velocity, and Paper.